src/Management/src/KubernetesCore/HostBuilderExtensions.cs

```diff
@@ -8,7 +8,7 @@
 using Steeltoe.Extensions.Logging;
 using Steeltoe.Management.Endpoint;
 
-namespace Steeltoe.Management.Kubernetes
+namespace Steeltoe.Management.KubernetesCore
```

Comment:
I'm not sure the namespace needs to change, I don't think we are including `Core` or `Base` in any of the namespaces